t/plugin/limit-count.t

```diff
@@ -1371,3 +1371,63 @@ GET /t
 qr/property \"time_window\" validation failed: expected 0 to be strictly greater than 0/
 --- no_error_log
 [error]
+
+
+
+=== TEST 43: set route, with rejected_msg
```

Comment:
Let's move the test to the new test file. See https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#check-code-style-and-test-case-style

> When the test file is too large, for example > 800 lines, you should split it to a new file. Please take a look at t/plugin/limit-conn.t and t/plugin/limit-conn2.t

Reply:
okay, thanks, done~